### PR TITLE
cookbook for adding new platform

### PR DESCRIPTION
**Pull request by dwicinas**

### CODE REVIEW

File: install_template/README.md [hunk 1]
```diff
@@ -225,3 +225,44 @@ In a handful of situations it is useful to employ conditionals to include or mod
...
+   -The number of topics that need to be updated will vary depending on the platform being added. For RHEL 9, two new topics were created: **rhel-9-or-ol-9.njk** and **almalinux-9-or-rocky-linux-9.njk**.
+
+1. Update **templates/platformBase/ppc64le_index.njk** to add an entry in the navigation block in the front matter. For example, for RHEL 9, this entry was added:
+   - {{productShortname}}_rhel_9
+
+1. Update **templates/platformBase/x86_64_index.njk** to add entires in the navigation block in the front matter. For example, for RHEL 9, these entries were added:
+   - {{productShortname}}_rhel_9
+   - {{productShortname}}_other_linux_9
+
+1. In the **templates/products** folder, for each platform, copy existing topics, such as **rhel-8-or-ol-8.njk**, and use the copied file create a new version.
```
Comment (nidhibhammar) [inline install_template/README.md:264]:
```suggestion
1. In the **templates/products** folder, for each product, copy existing topics, such as **rhel-8-or-ol-8.njk**, and use the copied file create a new version. 
```

File: install_template/README.md [hunk 2]
```diff
@@ -225,3 +225,44 @@ In a handful of situations it is useful to employ conditionals to include or mod
 Be wary of over-using these; prefer inheritance structures that push this content down to sufficiently-specific "leaf" templates instead. As a general rule, use conditionals only when the test is much, much simpler than the alternative inheritance structure, and be willing to abandon it when (over time) that simplicity is lost.
 
 In particular, avoid the trap of setting a flag in a leaf template and then checking it in a base template: this separates the context from the decision, making it extremely difficult to judge when the test has lost its value.
+
+#### Adding a new platform
+
+1. Modify **config.yaml**. For example, when adding the RHEL 9 platform, the following entries were made to each product:
```
Comment (djw-m) [inline install_template/README.md:231]: I'd like to see some precision here on what you add:

- name:
String: Name of the platform (or platforms) (Is there a reference set of names we work with? When adding a new version are there root names we should use?)
  arch: 
String: CPU Architecture (again, reference list of architecture names anywhere? What if its a new architecture (coming soon for example, aarch64)
  supported-versions:
What is this? A list? What precision of version number, what delimiter?

File: install_template/README.md [hunk 3]
```diff
@@ -225,3 +225,44 @@ In a handful of situations it is useful to employ conditionals to include or mod
...
+- name: AlmaLinux 9 or Rocky Linux 9
+   arch: x86_64
+   supported versions: [<supported_versions>]
+- name: RHEL 9 or OL 9
+   arch: x86_64
+   supported versions: [<supported_versions>]
+   
+1. In **templates/platformBase_deploymentConstants.njk**, update the `map_platform` and `map_platform_old` blocks. For example, for RHEL 9, the following lines were added to both blocks of code:
+
+"AlmaLinux 9 or Rocky Linux 9": "other_linux9",
```
Comment (djw-m) [inline install_template/README.md:245]: What's the difference between other_linux9 and rhel9? Again, where do these names come from?

File: install_template/README.md (hunk 2 above)
Comment (drothery-edb) [reply install_template/README.md:231]: Good suggestions, Dj. Nidhi, Dave, and I have discussed this a bit and we feel we should limit the scope in this PR, but try to add more information to the readme in a separate PR. Nidhi and I both thought that the general topic could be added to the agenda for your weekly meeting with Nidhi.

File: install_template/README.md (hunk 3 above)
Comment (drothery-edb) [reply install_template/README.md:245]: This will be a fun discussion point for you and Nidhi.

**Review by drothery-edb — APPROVED** [review]

**Pull request merged**